### ISSUE TITLE
Rotated Top XAxis labels gets out of view.

### ISSUE THREAD

**Issue by DawidvanGraan** [PhilJay/MPAndroidChart#1511]
Please see the Screenshot of the Top XAxis labels being cut off. No codes changes made when switching between Library Version `v2.2.0` (first image) and Library Version `v2.2.2` (Second Image).

_(not sure if related to #1462)_

```java
// Setup Axis
final XAxis xAxis = comboChart.getXAxis();
xAxis.setPosition(XAxis.XAxisPosition.TOP_INSIDE);
xAxis.setDrawLabels(true);
xAxis.setDrawAxisLine(false);
xAxis.setAxisLineColor(ActivityCompat.getColor(this, R.color.colorSecondaryText26));
xAxis.setDrawGridLines(true);
xAxis.setLabelsToSkip(0);
xAxis.setLabelRotationAngle(270f);
xAxis.setValueFormatter((original, index, viewPortHandler) -> {
  switch (index) {
      case 0:
          return getString(R.string.chart_label_05h00);
      case 16:
          return getString(R.string.chart_label_21h00);
  }
  return "";
});
```

![mpchart_bug](https://cloud.githubusercontent.com/assets/1670437/13315493/a3a051c6-dbb3-11e5-8119-0228a31ff6c1.png)

**Comment by PhilJay** [comment]
Thank you for your report. Unfortunately I cannot reproduce the issue. 
But it is highly likely this was (again) fixed past 2.2.2 - version 2.2.3 will be out soon I the issue should no longer be present then.

**Comment by DawidvanGraan** [comment]
@PhilJay Thanks for the reply! Wil keep an eye out for v2.2.3
